doc/source/release/1.15.0-notes.rst

```diff
@@ -17,96 +17,1447 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+  careful that your index arrays are 32-bit. We are working on 64bit support.
+- The vendored ``ARPACK`` library has been upgraded to version ``3.9.1``.
+- COO, CSR, CSC and LIL formats now support the ``axis`` argument for
+  ``count_nonzero``.
+- Sparse arrays and matrices may now raise errors when initialized with
+  incompatible data types, such as ``float16``.
+- ``min``, ``max``, ``argmin``, and ``argmax`` now support computation
+  over nonzero elements only via the new ``explicit`` argument.
 
 
 `scipy.spatial` improvements
 ============================
+- ``Rotation.concatenate`` now accepts a bare ``Rotation`` object, and will
+  return a copy of it.
 
 
 `scipy.special` improvements
 ============================
+- The factorial functions ``special.{factorial,factorial2,factorialk}`` now
+  offer an extension to the complex domain by passing the kwarg
+  ``extend='complex'``. This is opt-in because it changes the values for
+  negative inputs (which by default return 0), as well as for some integers
+  (in the case of ``factorial2`` and ``factorialk``; for more details,
+  check the respective docstrings).
+- `scipy.special.zeta` now defines the Riemann zeta function on the complex
+  plane.
+- `scipy.special.softplus` computes the softplus function
+- The spherical Bessel functions (`scipy.special.spherical_jn`,
+  `scipy.special.spherical_yn`, `scipy.special.spherical_in`, and
+  `scipy.special.spherical_kn`) now support negative arguments with real dtype.
+- `scipy.special.logsumexp` now preserves precision when one element of the
+  sum has magnitude much bigger than the rest.
+- The accuracy of several functions has been improved:
+
+  - `scipy.special.ncfdtr` and `scipy.special.nctdtr` have been improved
+    throughout the domain.
+  - `scipy.special.hyperu` is improved for the case of ``b=1``, small ``x``,
+    and small ``a``.
+  - `scipy.special.logit` is improved near the argument ``p=0.5``.
+  - `scipy.special.rel_entr` is improved when ``x/y`` overflows, underflows,
+    or is close to ``1``.
 
+- `scipy.special.gdtrib` may now be used in a CuPy ``ElementwiseKernel`` on
+  GPUs.
+- `scipy.special.ndtr` is now more efficient.
 
 `scipy.stats` improvements
 ==========================
+- A new probability distribution infrastructure has been added for the
+  implementation of univariate, continuous distributions:
 
+  - Speed, accuracy, and memory advantages of the new infastructure are
+    detailed in
+  - `scipy.stats.Normal` represents the normal distribution with the new
+    interface. In most cases, its methods are faster and more accurate than
+    those of `scipy.stats.norm`.
+  - Use `scipy.stats.make_distribution` to treat an existing continuous
+    distribution (e.g. `scipy.stats.norm`) with the new infrastructure.
+    This can improve the speed and accuracy of existing distributions,
+    especially for methods not overridden with custom formulas in the
+    implementation.
```

Comment:
#21746 added `scipy.stats.{exp,log}`

Reply:
#21752 added `truncate`, `abs`, and `Mixture`

Reply:
https://github.com/scipy/scipy/pull/21982/files#r1874575198